gravitee-apim-console-webui/src/management/api/creation-get-started/api-creation-get-started.component.ts

```diff
@@ -17,7 +17,7 @@
 import { Component, Inject, OnDestroy, OnInit } from '@angular/core';
 import { catchError, filter, switchMap, takeUntil, tap } from 'rxjs/operators';
 import { EMPTY, of, Subject } from 'rxjs';
-import { MatLegacyDialog as MatDialog } from '@angular/material/legacy-dialog';
+import { MatDialog } from '@angular/material/dialog';
```

Comment:
here we use a MatDialog but it line 87 it reference a component that has not been migrated. So the import is broken if we don't migrate both components. Can we do it in this PR before merging ?

Reply:
i will migrate it later. I've change to keep legacy dialog here